cluster-autoscaler/cloudprovider/gce/templates_test.go

```diff
@@ -137,13 +159,16 @@ func TestBuildNodeFromTemplateSetsResources(t *testing.T) {
 				assert.NotNil(t, node.Status)
 				assert.NotNil(t, node.Status.Capacity)
 				assert.NotNil(t, node.Status.Allocatable)
-				capacity, err := tb.BuildCapacity(tc.physicalCpu, tc.physicalMemory, tc.accelerators, OperatingSystemLinux, OperatingSystemDistributionCOS, -1, tc.pods)
+				if tc.isEphemeralStorageBlocked {
+					tc.physicalEphemeralStorage = 0
```

Comment:
nit: I don't think it changes anything, but modifying test case definition from within test feels inelegant. I would use a local variable instead.

Reply:
Done